### ISSUE TITLE
Ubuntu에서 Django migrate시 Access denied 오류가 뜨는 경우 

### ISSUE THREAD

**Issue by elijah0430** [wafflestudio/rookies#88]
한 곳에서 계속 막히는데 구글링으로 잘 찾지 못해서 Issue에 등록하였습니다. 
현재 Ubuntu 멀티부팅으로 설정해놓고, assignment 3번까지는 완료했습니다. 
4번 진행중인데, 
python3 manage.py migrate 명령어로 migrate을 시도하였습니다. 하지만 그럴 때마다 
django.db.utils.OperationalError: (1045, “Access denied for user ‘waffle-backend’@‘localhost’ (Using password: Yes)”)
라는메세지가 뜨면서 오류가 발생합니다 

혹시 몰라서 sudo로도 실행해 보았는데, 그 경우에는 Django 모듈을 인식하지 못하는 듯한 메세지가 떴습니다
(정확히는 “ImportError: Couldn’t import Django. Are you sure it’s installed and available on your PYTHONPATH environment variable? Did you forget to activate a virtual environment?” 입니다)

현재 가상환경은 활성화 해놓은 상태이며, requirements.txt의모듈도 모두 설치한 상태입니다. 
혹시 피드백 주실 수 있으시다면 정말 감사하겠습니다.

읽어주셔서 감사합니다.

**Comment by eldpswp99** [comment]
보통 access denied 오류는 유저네임/패스워드가 잘못되어서 발생합니다. db를 만들고 거기에 권한을 준 mysql 유저를 만드셨나요?

**Comment by elijah0430** [comment]
네... 혹시 몰라서 mysql -u -p 명령어로 로그인하여서 username과password가 맞는지 확인해보았는데, 맞는 것 같았습니다. 
구글링할 때는 보통 mysql grant 이슈라고 해서 grant all privileges on waffle_backend.* to ‘waffle_backend’@‘localhost’ 멍령어도 다시 시도해 보았는데, 차도가 없었습니다......

**Comment by eldpswp99** [comment]
flush privileges까지 하셨나요? 근데 지금 로그인 자체가 안되는거 같은데.. https://technote.kr/32 여기 참고하셔서 유저목록 스크린샷으로 찍어주실수 있을까요?

**Comment by elijah0430** [comment]
![스크린샷, 2020-09-03 13-45-48](https://user-images.githubusercontent.com/68674522/92072274-d8494780-edeb-11ea-9424-bb05075702e5.png)

**Comment by elijah0430** [comment]
이렇게 하면 될까요?

**Comment by eldpswp99** [comment]
유저이름이 지금 waffle_backend(언더스코어)로 되어있으신데 waffle-backend로 새로 만드셔서 해보시겠어요?

**Comment by davin111** [comment]
@elijah0430 라벨 달아주세용

**Comment by elijah0430** [comment]
> 유저이름이 지금 waffle_backend(언더스코어)로 되어있으신데 waffle-backend로 새로 만드셔서 해보시겠어요? 새로 만드시면 데이터베이스에 grant privileges다시 해주셔야 합니다.

허무할 정도로 잘 작동하네요....ㅠㅠ
앞으로는 이런 기본적인 사항들 꼼꼼히 확인해서 이런 일 없도록 하겠습니다
꼼꼼히 봐주셔서 감사합니다!!!! :)

**Comment by elijah0430** [comment]
> @elijah0430 라벨 달아주세용

제가 초대링크가 만료되기 전에 승인하지 않았어서 지금 라벨 달기가 안 되네요 ㅠㅠ 죄송합니다 조금 뒤에 달도록 하겠습니다!!
